.github/workflows/nightly_cron.yml

```diff
@@ -0,0 +1,79 @@
+name: Test against latest dependencies
+
+# Run if we change any of these paths and every night at 1 (or 2) AM Central time
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    paths:
+    - 'requirements.txt'
+    - '.github/workflows/nightly_cron.yml'
+
+  schedule:
+    - cron: '0 7 * * *'  # Every day at 07:00 UTC (1AM CST or 2AM CDT)
+
+jobs:
+  build:
+    name: Build against latest dependencies
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+        python-version: ['3.8']
+        architecture: ['x86', 'x64']
+        exclude:
+          # Don't build 32-bit on Mac or Linux
+          - os: macos-latest
+            architecture: x86
+
+          - os: ubuntu-latest
+            architecture: x86
```

Comment:
An aside, I really like the way GH actions structures their YAML. Very readable in plain English.

Reply:
Yeah, they make it pretty simple! Wish their docs were a little better though 🤔